ogcore/default_parameters.json

```diff
@@ -645,7 +645,7 @@
         }
     },
     "alpha_I": {
-        "title": "Exogenous fraction of government spending that goes towards investment in infrastructure (public capital)",
+        "title": "Government spending that goes towards investment in infrastructure (public capital) as a fraction of GDP",
```

Comment:
@jdebacker. I changed this description in [my PR](https://github.com/jdebacker/OG-Core/pull/24/) to your branch to be the following:
> "Exogenous fraction of GDP that goes towards government investment in infrastructure (public capital)"

I can have my PR to a good stopping place in about an hour. I just need to reverse the changes I made to the documentation and code regarding our discussion about the production function.